src/components/ControllerTop.vue

```diff
@@ -202,6 +208,11 @@ export default {
       }
     }
   },
+  async mounted() {
+    await this.initializeKeyboards();
```

Comment:
just make all these tasks synchronous, it doesn't make sense to not wait for all of these things to complete